ospf6d/ospf6_top.c

```diff
@@ -973,6 +974,7 @@ DEFUN (ospf6_distance,
 	VTY_DECLVAR_CONTEXT(ospf6, o);
 
 	o->distance_all = atoi(argv[1]->arg);
+	ospf6_restart_spf(o);
```

Comment:
Let's at least add a check to not trigger the restart when distance value is not actually changed.

Reply:
indeed! thanks, just changed....